packages/pluggableWidgets/carousel-web/src/Carousel.tsx

```diff
@@ -1,16 +1,19 @@
-import { createElement, ReactNode, useCallback, ReactElement } from "react";
-import { ValueStatus } from "mendix";
+import { createElement, ReactNode, useCallback, ReactElement, useEffect } from "react";
+import { ValueStatus, GUID, ObjectItem } from "mendix";
 import { executeAction } from "@mendix/piw-utils-internal";
 import { CarouselContainerProps } from "../typings/CarouselProps";
 import { Carousel as CarouselComponent } from "./components/Carousel";
 import loadingCircleSvg from "./ui/loading-circle.svg";
 import classNames from "classnames";
 import "./ui/Carousel.scss";
+import Swiper, { Navigation, Pagination, EffectFade, Autoplay } from "swiper";
 
 export function Carousel(props: CarouselContainerProps): ReactNode {
-    const { dataSource, content, showPagination, loop, tabIndex, navigation, animation, delay, autoplay } = props;
```

Comment:
why are we suddenly not destructuring `content` and `dataSource`? 🤔

Reply:
We were having issues with the design mode, where Swiper module was undefined, so we (me and Diego) started debug process, and in that process we changed the way how content and datasource are represented in the function. but now I guess it doesn't matter because our issue is solved. And it was not directly connected with the destructuring